docs/execution_environments.md

```diff
@@ -2,9 +2,9 @@
 
 All jobs use container isolation for environment consistency and security.
 Compliant images are referred to as Execution Environments (EE)s.
-For more information, see [ansible-runner EE docs](https://ansible-runner.readthedocs.io/en/latest/execution_environments.html)
-for information on how container isolation works, and see [ansible-builder docs](https://ansible-builder.readthedocs.io/en/latest/index.html)
-for instructions on how to build them.
+
+For more information about the EE technology as well as how to build and test EEs, see the
+[Getting started with Execution Environments guide](https://docs.ansible.com/ansible/devel/getting_started_ee/index.html). 
```

Comment:
We should also keep a link to the actual builder docs here imo.

Reply:
as in https://ansible.readthedocs.io/projects/builder/en/latest/

Reply:
I agree!